Luminary099/R30.agc

```diff
@@ -209,10 +210,10 @@ BOTHPAD		STCALL	RPADTEM
 			-TPER		# HPERMIN (300 OR 35) KFT.
 			TICKTIFF	# (-TPER = 0)
 TICKTPER	DLOAD	DAD		# (-TPER NON ZERO) TFF WAS NOT COMPUTED.
-			-TPER		# BUT WAS SET TO 59M59S.  DON'T DICK TFF, DO
+			-TPER		# BUT WAS SET TO 59M59S.  DON'T TICK TFF, DO
 			TSTART82	# TICK -TPER.  DISPLAY BOTH.
 		STORE	-TPER		# -TPER CORRECTED FOR TIME SINCE V82GOFF1
-		EXIT			# BEGAN
+		EXIT			# BEGAN.
 		CAF	BIT1
 		TS	V82FLAGS	# INFORMS TICKTEST TO INCREMENT ONLY -TPER
 		TC	ENDOFJOB
```

Comment:
![image](https://user-images.githubusercontent.com/3440094/95388362-15709000-08ea-11eb-9fb5-5f18eb083654.png)

2 lines below, double space before `TFF WAS COMPUTED` and no space before `TICK TFF`